compliance/elasticsearch/pom.xml

```diff
@@ -10,8 +10,8 @@
 	<name>RDF4J Elasticsearch Sail Tests</name>
 	<description>Tests for Elasticsearch.</description>
 	<properties>
-		<lucene.version>7.5.0</lucene.version>
-		<elasticsearch.version>6.5.4</elasticsearch.version>
+		<lucene.version>7.7.2</lucene.version>
+		<elasticsearch.version>6.8.8</elasticsearch.version>
```

Comment:
Can we just remove these properties, so that the versions are only delcared once, in the root pom?

Reply:
Good point, done